megamek/src/megamek/common/Crew.java

```diff
@@ -1173,20 +1252,6 @@ public void setCurrentPilot(int pos) {
         actedThisTurn[pos] = true;
     }
 
-    /**
-     * Set the gunner slot. If a multicrew cockpit uses the same crew member as both pilot and gunner
-     * (i.e. cockpit command console), sets the pilot as well.
-     *
-     * @param pos The slot index to set as gunner.
-     */
-    public void setCurrentGunner(int pos) {
```

Comment:
What happened to all these functions that got removed?

Reply:
Unused for years, so I removed them.